### ISSUE TITLE
P-GTID not showing up on topology of replication delayed host

### ISSUE THREAD

**Issue by tomkrouper** [openark/orchestrator#314]
I am not understanding why `host-delayed.dc3.example.com` in the example below is not showing `P-GTID`. This is preventing us from moving the delayed host below `host-replica1` without replication being stopped. Checking our chatops backlog it appears this is new behavior but from looking at the code around where `UsingPsuedoGTID` is set, it doesn't look like anything has changed in recent months. Please advise on why a delayed host may not be recognized as using P-GTID. Thanks.

```
$ orchestrator-client -c topology --alias somecluster
host-master.dc1.example.com:3306         [0s,ok,5.7.18-log,rw,ROW,>>,P-GTID]
+ host-delayed.dc3.example.com:3306       [14400s,ok,5.7.19-log,rw,ROW,>>]
+ host-replica1.dc1.example.com:3306      [0s,ok,5.7.19-log,ro,ROW,>>,P-GTID]
  + host-replica2.dc1.example.com:3306    [0s,ok,5.7.19-log,ro,ROW,>>,P-GTID]
  + host-replica3.dc1.example.com:3306    [0s,ok,5.7.19-log,ro,ROW,>>,P-GTID]
  + host-replica4.dc2.example.com:3306    [0s,ok,5.7.18-log,ro,ROW,>>,P-GTID]
    + host-replica5.dc2.example.com:3306  [0s,ok,5.7.18-log,ro,ROW,>>,P-GTID]
    + host-replica6.dc2.example.com:3306  [0s,ok,5.7.18-log,ro,ROW,>>,P-GTID]
      - host-testing.dc3.example.com:3306 [unknown,invalid,5.7.19-log,rw,nobinlog,downtimed]
```

**Comment by tomkrouper** [comment]
I set replication delay to zero and monitored the recovery and saw this happen...
```
+ host-delayed.dc3.example.com:3306     [1833s,lag,5.7.19-log,rw,ROW,>>]                                + host-delayed.dc3.example.com:3306     [1801s,lag,5.7.19-log,rw,ROW,>>]
+ host-delayed.dc3.example.com:3306     [1781s,lag,5.7.19-log,rw,ROW,>>,P-GTID]
+ host-delayed.dc3.example.com:3306     [1748s,lag,5.7.19-log,rw,ROW,>>,P-GTID]
```
Seems like it may only allow P-GTID for delays below 30 minutes.  Why is that?

**Comment by shlomi-noach** [comment]
The reason for this problem is a (bad) change I made in our internal `orchestrator` configuration file. I recently changed 

```
      "DetectPseudoGTIDQuery": "select count(*) as pseudo_gtid_exists from XXXXXX.pseudo_gtid_status where anchor = 1 and time_generated > now() - interval 2 day",
```

to

```
      "DetectPseudoGTIDQuery": "select count(*) as pseudo_gtid_exists from XXXXXX.pseudo_gtid_status where anchor = 1 and time_generated > now() - interval 30 minute",
```

To elaborate on what this is all about: when `orchestrator` is asked to relocate a replica it checks what its options are. GTID? Pseudo-GTID? Binlog server? Normal file:pos? etc.

We are using Pseudo-GTID, and you would expect `orchestrator` to know that.

Pseudo-GTID enries are injected into the binary logs, and `orchestrator` does not parse the binary logs all the time, only when it is actually running a Pseudo-GTID-based relocation. So how does `orchestrator` know a server has Pseudo-GTID in the first place?

When we inject P-GTID, we also update a table, not different from a `heartbeat` table. The `DetectPseudoGTIDQuery` is the query to check that table and confirm there's a P-GTID entry. Only my change said "only consider the entry if it's newer than 30 minutes", and of course this will not include our delayed replicas which are lagging a few hours behind.
